src/databricks/labs/ucx/account/workspaces.py

```diff
@@ -10,6 +11,21 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class AccountGroupDetails:
+    id: str
+    members: list[ComplexValue] | None = None
+
+
+@dataclass
+class AccountGroupCreationContext:
+    valid_workspace_groups: dict[str, Group]
+    created_groups: set[str] = field(default_factory=set)
```

Comment:
I don't think we need this, is it the same as the set of keys in `created_group_details`?